PhysicsTools/HeppyCore/python/framework/config.py

```diff
@@ -97,6 +104,12 @@ def build_name(self):
         name = '_'.join([class_name, self.instance_label])
         return name 
 
+    def clone(self, **kwargs):
```

Comment:
`clone(**kwargs)` is made to work as in the CMSSW config file creation, to provide a new instance of the module but with some parameters modified, i.e.

    module2 = module1.clone(
          someParameter = newValue
    )

I can provide an example in the docstring

The reason for localizing the `import copy` is that it's needed only internally in the implementation of the method, and so there's no need to expose it and risk possible conflicts if e.g. we will end up defining a copy function or similar.
I can move it at the top of the file if you insist